src/app/action-items/action-items.component.html

```diff
@@ -1,38 +1,42 @@
-<div *ngIf="loading" class="spinner">
+<div *ngIf="isloadingActionItems" class="spinner">
```

Comment:
**FIX:** Please add UI tests for that. We should generally test all directives like `ngIf` with UI tests, as this is conditional logic.